envs/requirements-sklearn.txt

```diff
@@ -0,0 +1,17 @@
+scikit-learn
```

Comment:
It probably makes sense to also specify the `numpy` and `scipy` packages in envs files.

Reply:
Numpy and scipy are deps of sklearn, and this env file is not supposed to be static.